packages/config/config/devices/0x0094/adc-sg130z.json

```diff
@@ -0,0 +1,28 @@
+{
+	"manufacturer": "Alarm.com",
+	"manufacturerId": "0x0094",
+	"label": "ADC-SG130Z",
+	"description": "Smart Gateway",
+	"devices": [
+		{
+			"productType": "0x0011",
+			"productId": "0x0111",
+			"zwaveAllianceId": [4151, 4304, 4309, 4310]
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 56,
```

Comment:
Woah, this seems excessive

Reply:
So what do you want me to do here? This is a gateway, so remove the section entirely?

Reply:
@AlCalzone Need direction on these

Reply:
From the manual "Association group one supports a maximum of 56 node associations."